docsrc/imap/reference/admin/locations.rst

```diff
@@ -35,6 +35,55 @@ In general, Cyrus allows one to maintain several separate partitions for
 each of these data types, and to establish rules governing distribution
 of data within each pool.
 
+Types of Partitions
+-------------------
+
+Cyrus supports several different types of partitions:
+
+*   Mail Spool Partitions
+*   Metadata Partitions
+*   Backup Partitions
+*   Archive Partitions
+*   Search Partitions
+
+Each of these are discussed in their own sections of the documentation.
+All share in common a few aspects in how they are configured.  For each
+partition defined, you must tell Cyrus where the partition is rooted in
+the filesystem.  This is accomplished via a "*partition*-*name*:" directive
+for each partition, where "*partition*" specifies the partition type and
+"*name*" is the actual name of the partition.
+
+Here are some sample declarations of each different type of partition
+supported within Cyrus.  For the purposes of this example, we'll
+stipulate the following:
+
+*   There are two main storage pools, "fast" and "slow," which are on
+    SSDs and traditional disks, respectively.
+*   The mailboxes are divided alphabetically, with A to M in one group
+    and N to Z in the other.
+
+.. parsed-literal::
+
+    # The Mail Spool Partitions
+    partition-atom: /var/spool/cyrus/fast/mail/atom/
+    partition-ntoz: /var/spool/cyrus/fast/mail/ntoz/
+
+    # The Metadata Partitions
+    metapartition-atom: /var/spool/cyrus/fast/meta/atom/
+    metapartition-ntoz: /var/spool/cyrus/fast/meta/ntoz/
+
+    # Backup Partition
```

Comment:
I think having a backup partition entry specified here is misleading.  This is for the backup service, which should generally not be on the same Cyrus instance as your regular mail partitions.

Also, the backup service is experimental in 3.0.x anyway, so I don't think we want to encourage cargo-culting of configuration for it.